taskvine/src/worker/vine_cache.c

```diff
@@ -118,7 +118,7 @@ void vine_cache_scan(struct vine_cache *c, struct link *manager)
 	char *cachename;
 	HASH_TABLE_ITERATE(c->table, cachename, f)
 	{
-		vine_worker_send_cache_update(manager, cachename, f->original_type, f->cache_level, f->size, f->mode, f->transfer_time, f->start_time);
```

Comment:
Could be make it so that the same info is sent back in the ready message?  In that way we don't have to change the protocol. Changing the protocol is always a bit of a headache when workers can't connect because of it.

Reply:
So the `cache-update` message itself didn't change, it still has the same format. I just cleaned up the function signature a bit.

What did change is that the manager now expects the worker to send a cache-update for all file types, where it didn't send them before.   That's the protocol change.

Reply:
Ah got it. Thanks for explaining it!